### PR TITLE
Scaffold React application

### PR DESCRIPTION
**Pull request by stefankarlberg**
# [PT STORY](https://www.pivotaltracker.com/story/show/166112708)
## Changes we propose in this PR:
* Scaffold new React application

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,68 @@
...
+
+This section has moved here: https://facebook.github.io/create-react-app/docs/advanced-configuration
+
+### Deployment
+
+This section has moved here: https://facebook.github.io/create-react-app/docs/deployment
+
+### `npm run build` fails to minify
+
+This section has moved here: https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify
```
Comment (Carrosen) [inline README.md:68]: Please delete this README since we have duplicates, and rename the other one(take away old) :)

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './App';
+import * as serviceWorker from './serviceWorker';
+
+ReactDOM.render(<App />, document.getElementById('root'));
+
+// If you want your app to work offline and load faster, you can change
```
Comment (Carrosen) [inline src/index.js]: Please remove comments :)

File: src/serviceWorker.js [hunk 3]
```diff
@@ -0,0 +1,135 @@
+// This optional code is used to register a service worker.
+// register() is not called by default.
+
+// This lets the app load faster on subsequent visits in production, and gives
+// it offline capabilities. However, it also means that developers (and users)
+// will only see deployed updates on subsequent visits to a page, after all the
```
Comment (Carrosen) [inline src/serviceWorker.js]: Same here

File: README.md (hunk 1 above)
Comment (zanenkn) [reply README.md:68]: yes

File: src/index.js (hunk 2 above)
Comment (zanenkn) [reply src/index.js]: yes

File: src/serviceWorker.js (hunk 3 above)
Comment (zanenkn) [reply src/serviceWorker.js]: okej

**Pull request merged**